packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx

```diff
@@ -79,6 +81,28 @@ export class ESService_LegendStudioPlugin
     ];
   }
 
+  getExtraElementIconGetters(): ElementIconGetter[] {
+    return [
+      (type: string): React.ReactNode | undefined => {
+        if (type === SERVICE_STORE_ELEMENT_TYPE) {
+          return (
```

Comment:
We genrally don't set color background for our icon so to be consistent let's remove this. Also, we create a generic color for the icon, see `--color-class` for example. :) Honestly, I'm just trying to be consistent here, but I might (likely) regret saying this you know, knowing that we will soon revamp our styling/theming system in the future :sweat:

Reply:
@akphi so I requested @gayathrir11 to add this background color to make the icon close to what swagger has so it's easily recognizable. I am totally fine if you believe it's not the best strategy, just wanted to mention this.

Reply:
understood but no